lib/glimesh/chat.ex

```diff
@@ -192,6 +192,20 @@ defmodule Glimesh.Chat do
   """
   def get_chat_message!(id), do: Repo.get!(ChatMessage, id) |> Repo.preload([:user, :channel])
 
+
+  @doc """
+  Gets the latest follower message from a channel for a user
+  """
+  def get_follow_chat_message_for_user(channel, user) do
+    Repo.all(
```

Comment:
Could this be .one?

Reply:
Yes :)
```suggestion
    Repo.one(
```
I wasn't fully sure how I was gonna do it, plus I did a fat copy-paste of the function higher up